docs/Reference/CLI/Subcommands/Slashing-Protection.md

```diff
@@ -165,8 +165,8 @@ Repairs corrupted slashing-protection data files used by Teku.
     ```
 
 Reads and reports potential slashing-protection file problems, but doesn't update any files.
-You can specify which files are checked using [`--config-file`](#config-file_2), [`--data-base-path`](#data-base-path-data-path),
-or [`--network`](#network).
+You can specify which files are checked using [`--config-file`](#config-file_2), [`--data-base-path`](#data-base-path-data-path).
+Adding [`--network`](#network) will get the genesis time for the specified network if you need to calculate a slot.
```

Comment:
```
➜  teku slashing-protection repair --data-base-path teku/path --check-only-enabled
Computed mainnet slot 1968386, which will set attestation source/target to epoch 61512
done
➜  teku slashing-protection repair --data-base-path teku/path --check-only-enabled --network prater
Computed prater slot 1161391, which will set attestation source/target to epoch 36293
done
```
so above the 'computed XXX' is because '--network' defaults to mainnet, so initially it computes mainnet, then when i tell it I'm on prater, it computes that instead...

We can put network here, but what it does is compute the slot, which is based on genesis time...

Reply:
Is this more accurate?

>Calculate the current slot by using [`--network`](#network) for the specified network, based on the genesis time.

Reply:
`Adding --network will allow the current slot to be calculated based on network configuration.` ?

Either that or maybe just remove it given that the other 2 parts of this help are discussing paths?